src/dotnet/commands/dotnet-install/InstallCommandParser.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class InstallCommandParser
+    {
+        public static Command Install()
+        {
+            return Create.Command(
+                "install", "",
+                Accept.NoArguments(), CommonOptions.HelpOption(), InstallTool());
+        }
+
+        private static Command InstallTool()
+        {
+            return Create.Command("tool",
+                "Install tool",
+                Accept.ExactlyOneArgument(o => "packageId")
+                    .With(name: "packageId",
+                        description: "Package Id in NuGet"),
```

Comment:
@KathleenDollard all the `name` and `description` are customer facing string. They are shown in --help